### ISSUE TITLE
Feature Request: permit multiple CustomActions (OnNodeUpdated, OnNodeConfigured, OnNodeStart)

### ISSUE THREAD

**Issue by nyetsche** [aws/aws-parallelcluster#5022]
The current code only permits a single script (with a list of args) for custom actions, as show in this example from the documentation:

```
CustomActions:
  OnNodeStart:
    Script: string
    Args:
      - string
  OnNodeConfigured:
    Script: string
    Args:
      - string
   OnNodeUpdated:
   Script: string
   Args:
     - string
```

We feel we could improve our workflow by specifying multiple actions, to be executed serially, with any one of them failing causing the Stack to fail. That'd require a move to a list, not keys.

Hypothetical example:

```
OnNodeConfigured:
  - Script: s3://add-users/script.sh
    Args:
      - "UserA"
      - "UserB"
  - Script: s3://install-spack/script.sh
```

There would have to be some extra logic to not break backwards compatibility, since all previous configurations would have just the single `Script` key. I'm open to suggestions other than turning `Script` into a list.

**Comment by enrico-usai** [comment]
Hi @nyetsche thanks for opening this issue.

We are aware of the limitation and we already started working on it: https://github.com/aws/aws-parallelcluster-cookbook/pull/1895

**Comment by enrico-usai** [comment]
Feature released with 3.6.0: https://github.com/aws/aws-parallelcluster/releases/tag/v3.6.0
